Add and update documentation image descriptions (#5)
isabela-pf/napari: docs/tutorials/fundamentals/installation.md

An empty napari viewer should appear as follows.

> ![image: An empty napari viewer ](../assets/tutorials/launch_cli_empty.png)

Reviewer: ```suggestion
![macOS desktop with a napari viewer window without any image opened in the foreground, and a terminal in the background with the appropriate conda environment activated (if applicable) and the command to open napari entered.](../assets/tutorials/launch_cli_empty.png)
```

this:

> ![image: release bundle

Reviewer: ```suggestion
```

> assets ](../assets/tutorials/installation/bundle_assets.png)

Reviewer: ```suggestion
[Cropped screenshot from GitHub with the Assets section (or "tab") expanded, containing links to download the app in the form of zip files for Linux, macOS, Windows, in addition to other links.](../assets/tutorials/installation/bundle_assets.png)
```

by dragging the `napari` icon onto the `Applications` icon.

> ![image: MacOS install to

Reviewer: ```suggestion
```

> Applications ](../assets/tutorials/installation/bundle_install.png)

Reviewer: ```suggestion
![Finder window of the napari bundle dmg file, containing the napari app icon and a shortcut icon to the Applications folder.](../assets/tutorials/installation/bundle_install.png)
```

After you drag the `napari` icon onto the `Applications` icon napari will be
installed in your Applications folder and available for opening by double
clicking on it.

> ![image: MacOS Applications folder with

Reviewer: ```suggestion
```

> napari ](../assets/tutorials/installation/bundle_applications_folder.png)

Reviewer: ```suggestion
![Finder window of the Applications folder containing the napari app.](../assets/tutorials/installation/bundle_applications_folder.png)
```
